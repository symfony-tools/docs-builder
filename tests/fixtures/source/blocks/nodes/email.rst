Lorem ipsum dolor sit amet user@example.com consectetur adipisicing elit.

Lorem ipsum dolor sit amet **user@example.com** consectetur adipisicing elit.

Lorem ipsum dolor **sit amet user@example.com consectetur** adipisicing elit.

Lorem ipsum dolor sit amet *user@example.com* consectetur adipisicing elit.

Lorem ipsum dolor *sit amet user@example.com consectetur* adipisicing elit.
