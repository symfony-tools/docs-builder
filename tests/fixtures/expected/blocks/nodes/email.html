<p>Lorem ipsum dolor sit amet <a href="mailto:user@example.com" class="reference internal">user@example.com</a> consectetur adipisicing elit.</p>

<p>Lorem ipsum dolor sit amet <strong><a href="mailto:user@example.com" class="reference internal">user@example.com</a></strong> consectetur adipisicing elit.</p>

<p>
    Lorem ipsum dolor
    <strong>sit amet <a href="mailto:user@example.com" class="reference internal">user@example.com</a> consectetur</strong>
    adipisicing elit.
</p>

<p>Lorem ipsum dolor sit amet <em><a href="mailto:user@example.com" class="reference internal">user@example.com</a></em> consectetur adipisicing elit.</p>

<p>
    Lorem ipsum dolor
    <em>sit amet <a href="mailto:user@example.com" class="reference internal">user@example.com</a> consectetur</em>
    adipisicing elit.
</p>
